json.h

```diff
@@ -266,6 +266,7 @@ json_value * json_parse_ex (json_settings * settings,
 
 void json_value_free (json_value *);
 
+int trailing_garbage (const json_char *);
```

Comment:
Does this function need to be declared in `json.h`? I don't think so, it should be a static function of `json.c`.